.eslintrc

```diff
@@ -34,7 +34,6 @@
     "jest/no-identical-title": "error",
     "jest/prefer-to-have-length": "warn",
     "jest/valid-expect": "error",
-    "valid-jsdoc": "off",
```

Comment:
I'm removing jsdoc, has been mostly unused and we can document components in better ways.